streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -667,7 +674,7 @@ public void validateCopartition() {
     private void validateGlobalStoreArguments(final String sourceName,
                                               final String topic,
                                               final String processorName,
-                                              final ProcessorSupplier<?, ?> stateUpdateSupplier,
+                                              final ProcessorSupplier<?, ?, ?, ?> stateUpdateSupplier,
```

Comment:
Could we use `Void, Void` instead?

Reply:
Sure; all we do it verify it's not null, but it doesn't hurt.